tests/file_uploads/urls.py

```diff
@@ -14,5 +14,6 @@
     path('getlist_count/', views.file_upload_getlist_count),
     path('upload_errors/', views.file_upload_errors),
     path('filename_case/', views.file_upload_filename_case_view),
+    path('stop_upload/', views.file_upload_stopped),
```

Comment:
```suggestion
    path('upload_stopped/', views.file_upload_stopped),
```

Reply:
I thought it would be better for the url to suggest an action rather than a result.